File: WOT-iOS/WOTData/WOTData/model/subclasses/UserSession+CoreDataProperties.m
//
//  UserSession+CoreDataProperties.m
//  WOTData
//
//  Created on 8/28/18.
//  Copyright © 2018. All rights reserved.
//
//

#import "UserSession+CoreDataProperties.h"

@implementation UserSession (CoreDataProperties)

+ (NSFetchRequest<UserSession *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"UserSession"];
}

@dynamic access_token;
@dynamic accound_id;
@dynamic currentSession;
@dynamic expires_at;
@dynamic isCurrent;
@dynamic nickname;

@end

File: WOT-iOS/WOTApi/WOTApi/response/managedObjects/autogenerated/UserSession+CoreDataProperties.swift
//
//  UserSession+CoreDataProperties.swift
//  WOTData
//
//  Created by Pavel Yeshchyk on 5/1/20.
//  Copyright © 2020 Pavel Yeshchyk. All rights reserved.
//
//

import CoreData

public extension UserSession {
    @nonobjc class func fetchRequest() -> NSFetchRequest<UserSession> {
        return NSFetchRequest<UserSession>(entityName: "UserSession")
    }

    @NSManaged var access_token: String?
    @NSManaged var accound_id: String?
    @NSManaged var currentSession: NSNumber?
    @NSManaged var expires_at: NSNumber?
    @NSManaged var isCurrent: NSNumber?
    @NSManaged var nickname: String?
}
